### DISCUSSION TITLE
Integration: Linux

### DISCUSSION THREAD

**Discussion by brendanfalk** [aws/amazon-q-developer-cli#38] [General]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Feature Details

Nuff said. 

Please upvote to follow our progress!

**Comment by Aneeqasif** [comment]
any updates here?

**Comment by aqwasch** [comment]
how is the progress?

**Comment by SolDavidCloud** [comment]
Really wanted

**Reply by NSTHEHACKER** [reply]
you can download it form github ? but i can't

**Comment by ansuff** [comment]
Maybe following [this](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-autocomplete-ssh.html) could work locally? I am testing it rn

**Reply by grant0417** [reply]
This will work for just the `q` executable, but will not enable Linux desktop support for autocomplete.

**Reply by ansuff** [reply]
Yeah, I keep getting frustrated bit by bit with this one!

**Comment by grant0417** [comment]
We have launched Amazon Q for command line on Linux, see the docs here to learn how to install: https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-installing.html#command-line-installing-ubuntu
